sdk/src/main/java/com/qonversion/android/sdk/internal/common/mappers/UserPropertiesMapper.kt

```diff
@@ -0,0 +1,10 @@
+package com.qonversion.android.sdk.internal.common.mappers
+
+internal class UserPropertiesMapper : Mapper<List<String>> {
```

Comment:
Seems like it's not UserProperties mapper but something like UserPropertiesResponse mapper?